### DISCUSSION TITLE
Example for Brutus under Amuse

### DISCUSSION THREAD

**Discussion by a-fahim** [amusecode/amuse#962] [Q&A] [answered]
Dear Experts,
I'm looking for a short example using Brutus Integrator (under Amuse package) to study the dynamics of three bodies.
However, I was not able to find such an example. Can you please help me to find or write something for this purpose?
Thanks

**Comment by spzwart** [comment, marked as answer]
dear Ali,

please, just take one of the minimal gravity examples, such as
gravity_minimal.py and replace the adopted integrator for Brutus.
The main difference between Brutus and most other N-body codes is the
possibility to set the numerical precision.
please, check out the AMUSE book for details.

cheers,
Simon

**Reply by rieder** [reply]
How would you specify the precision in Brutus?
